generators/cypress/templates/src/test/javascript/cypress/integration/administration/administration_spec.ts.ejs

```diff
@@ -0,0 +1,85 @@
+import {
+    <%_ if (!skipUserManagement) { _%>
+    userManagementPageHeadingSelector,
+    <%_ } _%>
+    metricsPageHeadingSelector,
+    healthPageHeadingSelector,
+    logsPageHeadingSelector,
+    configurationPageHeadingSelector,
+} from '../../support/commands';
+
+describe('/admin', () => {
+
+    <%_ if (authenticationType === 'oauth2') { _%>
+    beforeEach(() => {
+        cy.clearCache();
+        cy.kcLogout();
+        cy.visit('/');
+        cy.kcLogin('user');
+        cy.clickOnLoginItem();
+    });
+
+    after(() => {
+        cy.kcLogout();
+    });
+    <%_ } else { _%>
+    before(() => {
+        cy.visit('');
+        cy.login('admin','admin');
+    });
+
+    after(() => {
+        cy.clickOnLogoutItem();
+    });
+    <%_ } _%>
+
+    <%_ if (!skipUserManagement) { _%>
+    describe('/user-management', () => {
+        it('should load the page', () => {
+          cy.clickOnAdminMenuItem('user-management');
+          cy.get(userManagementPageHeadingSelector).should('be.visible');
+        });
+    });
+    <%_ } _%>
+
+    describe('/metrics', () => {
+        it('should load the page', () => {
+        <%_ if (clientFramework === "vue") { _%>
+        cy.clickOnAdminMenuItem('jhi-metrics');
+        <%_ } else { _%>
+        cy.clickOnAdminMenuItem('metrics');
+        <%_ } _%>
+        cy.get(metricsPageHeadingSelector).should('be.visible');
+        });
+    });
+
+    describe('/health', () => {
+        it('should load the page', () => {
+        <%_ if (clientFramework === "vue") { _%>
+        cy.clickOnAdminMenuItem('jhi-health');
+        <%_ } else { _%>
+        cy.clickOnAdminMenuItem('health');
+        <%_ } _%>
+        cy.get(healthPageHeadingSelector).should('be.visible');
+        });
+    });
+
+    describe('/logs', () => {
+        it('should load the page', () => {
+        cy.clickOnAdminMenuItem('logs');
+        cy.get(logsPageHeadingSelector).should('be.visible');
+        });
+    });
+
+    describe('/configuration', () => {
+        it('should load the page', () => {
+        <%_ if (clientFramework === "vue") { _%>
```

Comment:
it would be very cool to remove any framework specific condition (maybe in a next PR)